src/calcurse.c

```diff
@@ -48,18 +48,13 @@ int count, reg;
  */
 static void do_storage(int day_changed)
 {
+	/* * Save the selected item before rebuilding the day vector. */
```

Comment:
Does this extra `*` have any purpose? Also, I see why you moved that comment there but it is now less prominent than it was before.

Reply:
This was corrected by commit https://github.com/lfos/calcurse/pull/193/commits/7a137e98a8696eb9dd11cfe756a2cd996de3085b.